software/controller/lib/debugvars/vars_base.cpp

```diff
@@ -32,15 +32,22 @@ Base::Base(Type type, const char *name, Access access, const char *units, const
 
 const char *Base::name() const { return name_; }
 
-void Base::prepend_name(const char *prefix) {
+void Base::prepend_name(const char *prefix, bool strict) {
```

Comment:
I would maybe explain in a doxygen block what "strict" implies. What is expected behaviour with and without it.